Example/FLEXample/MiscNetworkRequests.m

```diff
@@ -153,7 +153,7 @@ - (void)sendExampleNetworkRequests:(NSURLSession *)sessionWithDelegate {
 }
 
 - (void)sendExampleWebsocketTraffic:(NSURLSession *)sessionWithDelegate {
-    NSString *APIKey = @"oCdCMcMPQpbvNjUIzqtvF1d2X2okWpDQj4AwARJuAgtjhzKxVEjQU6IdCjwm";
+    NSString *APIKey = @"VCXCEuvhGcBDP7XhiJJUDvR1e1D3eiVjgZ9VRiaV";
```

Comment:
Why did you change the API key? Did the old one expire or something?

Reply:
Yes, it returns error `{"error":"Invalid API key"}`.
Based on piesocket's website, its changed every week:

![image](https://user-images.githubusercontent.com/2391557/183832838-338a548f-ec7c-4d36-9e85-396f477a9410.png)